src/hsmm_mvpy/models.py

```diff
@@ -89,6 +90,12 @@ def __init__(self, data, eeg_data=None, sfreq=None, offset=0, cpus=1, event_widt
         else: self.template = template
         self.events = self.cross_correlation(data.data.T)#adds event morphology
         self.max_d = self.durations.max()
+        self.em_method = em_method
+        if self.em_method == "mean":
```

Comment:
shouldn't there be an elif for "max" as well?

Reply:
No because the max method doesn't need this new data representation

Reply:
As a result, using max method is also lighter in terms of memory